_build/copy-plugins.mjs

```diff
@@ -0,0 +1,47 @@
+import fs from "fs/promises";
+import path from "path";
+import { fileURLToPath } from "url";
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+
+const sourcePath = path.resolve(__dirname, "../node_modules/prismjs/src/plugins");
+const destPath = path.resolve(__dirname, "../plugins");
+
+async function copy () {
+	// We need { recursive: true } so the script doesn't fail if the folder already exists
+	await fs.mkdir(destPath, { recursive: true });
+
+	let plugins = await fs.readdir(sourcePath, { withFileTypes: true });
+	for (let plugin of plugins) {
+		if (!plugin.isDirectory()) {
+			continue;
+		}
+
+		let source = path.join(sourcePath, plugin.name);
+		let dest = path.join(destPath, plugin.name);
+		await fs.mkdir(dest, { recursive: true });
+
+		let files = await fs.readdir(source, { withFileTypes: true });
+		for (let file of files) {
+			if (!file.isFile()) {
+				continue;
+			}
+
+			let name = path.parse(file.name).name;
+			// Don't copy the plugin source files
```

Comment:
What if instead of excluding files based on heuristics we only include `README.md` and `demo.*`?

Reply:
My first attempt was exactly what you proposed. However, later, I realized that plugin docs might depend on other files, such as external stylesheets, scripts, images, data files, etc., and we don't know what names they might have. The only thing we can be sure of, though, is that we don't want to copy the plugin source files whose names happen to end with the plugin ID. So, I decided to follow that route.

Reply:
They shouldn't though. And if they do, they can always include them via HTML in `README.md`. I think it's okay to restrict what plugins can do in their docs a bit if it leads to a more predictable structure and less clutter in the source repo.